src/views/product-landing/components/heading/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { HeadingProps } from './types'
+import clamp from 'lib/clamp'
+import s from './style.module.css'
+
+function Heading({
```

Comment:
[Not to be addressed in this PR]: What do you think about moving this component to the main components folder and using it as a custom component for the docs content like our new docs-anchor component? 💭

Reply:
At a high level, on board with it! I think one challenge here that I'm always wary of is whether this holds true for design... ie will design be 100% consistent with these heading styles across landing pages and docs pages, or will they maybe want more flexibility on landing pages. However I think I need to temper that wariness a bit now that we have this stuff coming from the DS team, with the clear intent of being fully consistent 🙌 So very open to moving this into the main components folder in a follow-up PR! 💯